components/org.wso2.carbon.identity.application.authz.xacml/pom.xml

```diff
@@ -102,6 +102,10 @@
                             org.wso2.carbon.identity.entitlement.common.dto;version="${carbon.identity.framework.import.version.range}",
                             org.wso2.carbon.identity.entitlement.common.util;version="${carbon.identity.framework.import.version.range}",
                             org.jaxen; version="${axiom.org.jaxen.imp.pkg.version.range}",
+                            org.wso2.balana.utils.Constants; version="${balana.imp.pkg.version.range}",
```

Comment:
Usually explicit classes are not imported. Shall we import org.wso2.balana.utils instead of org.wso2.balana.utils.Constants